.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: elixir
+
+install:
+  - mix local.rebar --force
+  - mix local.hex --force
+  - mix deps.get
+  - mix compile
```

Comment:
Dialyzer is being run in dev environment, as well as inch.report task. So it would compile anyway